docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -69,6 +69,8 @@ In sbt:
 
 The above ensures that the Service Locator knows about the `weather` service. Then, if you need a Lagom service to communicate with it, simply `@Inject` the [`ServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html) and use it to either locate the `weather` service's URI, or perform some arbitrary work with it.
 
+Please Note : `lagomUnmanagedServices`  this only works in dev mode , for prod mode read [`lagom in production`](https://www.lagomframework.com/documentation/1.5.x/java/ProductionOverview.html)
```

Comment:
Hi @SahilAggarwalG, thanks for this improvement.

The section you edited is on a page of the docs under the `Dev Mode` section. I understand by your edit that it wasn't clear the details on this page only applied to dev mode. If that's the case, I think the change should be more prevalent (not apply to _only_  `lagomUnmanagedServices`). WDYT?

Also, please note in lagom the docs are duplicate between the `java/` and the `scala/` because there was just too many specific details to keep all the writeup in a single `.md` file for each page. I suggest we iterate your changes on the `docs/manual/java/guide/devmode/ServiceLocator.md` and, when ready, Id' ask you to also apply the changes to `docs/manual/scala/guide/devmode/ServiceLocator.md`.

Thanks!

Reply:
ok i would be fine